.eleventy.js

```diff
@@ -40,11 +40,44 @@ module.exports = function (eleventyConfig) {
   // eslint-disable-next-line no-unused-vars
   const live_packages = Object.entries(data.packages).map(([id, pkg]) => pkg).filter(pkg => !pkg.removed);
 
+  // if readme is in pkg
+  // transform some links
+  // https://raw.githubusercontent.com/relikd/CUE-Sheet_sublime/main/README.md
+  // => https://github.com/relikd/CUE-Sheet_sublime/blob/main/README.md
+  //
+  // https://gitlab.com/patopest/Sublime-Text-Cuelang-Syntax/-/raw/master/README.md
+  // => https://gitlab.com/patopest/sublime-text-cuelang-syntax/-/blob/master/README.md
+  //
+  // https://bitbucket.org/JeisonJHA/sublime-delphi-language/raw/master/README.md
+  // => https://bitbucket.org/JeisonJHA/sublime-delphi-language/src/master/README.md
+  //
+  // and store the under readme_url
   eleventyConfig.addCollection("packages", () => {
-    return live_packages.map(pkg => ({
-      ...pkg,
-      ...minimalPackage(pkg)
-    }));
+    return live_packages.map(pkg => {
+      let readme_url = pkg.readme;
+      if (typeof readme_url === 'string') {
+        // GitHub raw to blob
+        readme_url = readme_url.replace(
+          /^https:\/\/raw\.githubusercontent\.com\/([^/]+)\/([^/]+)\/([^/]+)\/(.+)$/,
+          'https://github.com/$1/$2/blob/$3/$4'
+        );
+        // GitLab raw to blob
+        readme_url = readme_url.replace(
+          /^https:\/\/gitlab\.com\/([^/]+)\/([^/]+)\/-\/raw\/([^/]+)\/(.+)$/,
+          'https://gitlab.com/$1/$2/-/blob/$3/$4'
+        );
```

Comment:
Gitlab sadly supports many nested project / groups. For example the following is a valid Gitlab URL where `trmnl` `test` `test` are all groups and then `thecrawl` is a project

Valid -> `https://gitlab.com/trmnl/test/test/thecrawl/-/raw/main/README.md`